### DISCUSSION TITLE
Any idea why podman desktop windows airgap binary is found suspicious

### DISCUSSION THREAD

**Discussion by tonymakt** [podman-desktop/podman-desktop#5665] [General]
https://www.virustotal.com/gui/url/25796e3fbb905f0d0399895103402b6e022f9698fd0916ea384d1d93540cbd72?nocache=1

According to ArcSight Threat Intelligence, it is classified as suspicious.
Even Censors Have a Backup: Examining China’s Double HTTPS Censorship Middleboxes - according to source ArcSight Threat Intelligence - 4 months ago

Any clue on this?

**Comment by benoitf** [comment]
you provided a link to the binary and not the binary itself

so basically it's telling you that github.com is a website with high traffic and it might cause problem if you're in China as traffic might be limited, etc

you'll see the same result if you just check for https://github.com

https://www.virustotal.com/gui/url/09a8b930c8b79e7c313e5e741e1d59c39ae91bc1f10cdefa68b47bf77519be57

**Reply by tonymakt** [reply]
Thanks 👍🏻I could finally proceed the application to introduce podman desktop into company trust list
